app/views/guides/community/how_to_ask.html.erb

```diff
@@ -142,6 +142,6 @@
     </p>
   <% end %>
 
-  <%= render 'static_pages/dashboard_steps/github_link', filename: 'app/views/static_pages/how_to_ask.html.erb' %>
+  <%= render 'guides/community/dashboard_steps/github_link', filename: 'app/views/guides/community/how_to_ask.html.erb' %>
```

Comment:
The first argument is the name of the view partial we use for github linke. We just need to update the filename value we're passing to it as an argument:

```suggestion
  <%= render 'static_pages/dashboard_steps/github_link', filename: 'app/views/guides/community/how_to_ask.html.erb' %>
```

This should get all the tests passing again.